stride/utils/fullwave.py

```diff
@@ -0,0 +1,295 @@
+
+import numpy as np
+import struct
+import mosaic
+
+
+__all__ = ['read_vtr_model3D', 'read_observed_ttr', 'read_signature_ttr',
+            'read_signature_txt', 'read_geometry_pgy']
+
+
+def read_vtr_model3D(vtr_path, swapaxes=False):
+    '''
+    Function to read 3D .vtr low kernel
+    binary data (model files) and returns the data as an as ndarray.
+
+    Code adapted from Oscar Calderon from Imperial College London's
+    FULLWAVE consortium.
+
+    Clues as to the structure of the binary file were provided by
+    Oscar Agudo from Imperial College of London's FULLWAVE consortium
```

Comment:
These two lines are not needed having the one above

Reply:
See commit [4561726](https://github.com/trustimaging/stride/pull/30/commits/4561726a69d476df160dfcfe4964b51b81859c37)